checkout-ui-custom/checkout6-custom.js

```diff
@@ -286,7 +286,8 @@ const MAX_TIME_EXPIRATION = 1000 * 60 * 5 // 5 minutes
       const checkShipping = setInterval(function () {
         if (
           b2bCheckoutSettings &&
-          b2bCheckoutSettings.permissions.includes(canEditAddress)
+          b2bCheckoutSettings.permissions.includes(canEditAddress) &&
+          !window.vtexjs.checkout.orderForm.loggedIn
```

Comment:
I'm not sure I understand, I thought this would be an OR instead of an AND. Now the checkout settings will only ever get cleared when the user is not logged in, is that what you want here? I thought it was not getting cleared before for logged out users, but if that's the case this code still won't clear it due to the && operation.

Reply:
Thinking of this, yes this is correct. I'll make the change.